docs/content/posts/2023-08-03-connector-updates.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "DynamoDB, MongoDB and OpenSearch externalized connectors"
+date: "2023-08-03T12:00:00Z"
+authors:
+- elphastori: 
+  name: "Elphas Toringepi"
+  twitter: "elphastori"
+aliases:
+- /news/2023/08/03/externalized-connectors.html
+---
+
+## New connectors
+
+We're excited to announce that Apache Flink now includes 3 new connectors for [DynamoDB](https://aws.amazon.com/dynamodb), [MongoDB](https://www.mongodb.com/) and [OpenSearch](https://opensearch.org/)! The connectors are available for both the [DataStream](https://nightlies.apache.org/flink/flink-docs-master/docs/connectors/datastream/overview/) and [Table/SQL](https://nightlies.apache.org/flink/flink-docs-master/docs/connectors/table/overview/) APIs.  
+
+- **[DynamoDB sink](https://nightlies.apache.org/flink/flink-docs-master/docs/connectors/datastream/dynamodb/)** - The DynamoDB sink writes to [Amazon DynamoDB](https://aws.amazon.com/dynamodb) using the [AWS v2 SDK for](https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/home.html) Java.
+- **[MongoDB connector](https://nightlies.apache.org/flink/flink-docs-master/docs/connectors/datastream/mongodb/)** - Flink now provides a [MongoDB](https://www.mongodb.com/) connector for reading and writing data from and to MongoDB collections with at-least-once guarantees.
```

Comment:
@leonardBang / @Jiabao-Sun do you want to add anything here?

Reply:
Thanks @dannycranmer, I think it's fine here.